android-sdk/src/main/java/org/sagebionetworks/bridge/android/di/ApplicationModule.java

```diff
@@ -37,4 +45,16 @@ public ApplicationModule(Context context) {
     Context getApplicatonContext() {
         return applicationContext;
     }
+    
+    @Provides
+    SocketFactory getSocketFactory() {
```

Comment:
Just curious, what is the purpose of this addition?

Reply:
https://github.com/square/okhttp/issues/3537
On Android O, now seeing this in logs
08-23 14:41:03.439 E/StrictMode: null
                                 java.lang.Throwable: Untagged socket detected; use TrafficStats.setThreadSocketTag() to track all network usage
                                     at android.os.StrictMode.onUntaggedSocket(StrictMode.java:2010)
                                     at com.android.server.NetworkManagementSocketTagger.tag(NetworkManagementSocketTagger.java:78)
                                     at libcore.io.BlockGuardOs.tagSocket(BlockGuardOs.java:47)
                                     at libcore.io.BlockGuardOs.socket(BlockGuardOs.java:310)
                                     at libcore.io.IoBridge.socket(IoBridge.java:667)